### ISSUE TITLE
SonLVL making Windows detect a Trojan

### ISSUE THREAD

**Issue by PumpkinMasterz** [sonicretro/SonLVL#123]
I was using SonLVL normally then today, it disappeared. I redownloaded SonLVL, then Windows detected the Trojan "Wacatac.B!ml" when I downloaded it again from the Updater. It also identified the same Trojan when trying to launch it and won't even launch it because Windows say's it contains a virus. Then SonLVL.exe deletes itself. Is this a false positive?

![Screenshot (119)](https://user-images.githubusercontent.com/110878079/183559291-2a6a3f8b-6355-4d2f-a903-543a549c900d.png)

**Comment by Caverns4** [comment]
Yes, it's a false positive. Windows 10 (and probably 11) just don't like the was SonLvl accesses stuff.
The only thing you can do is make an exception for the folder SonLvl and the updater are in.
